package.json

```diff
@@ -128,6 +142,7 @@
     "electron-builder": "19.48.3",
     "electron-mocha": "^5.0.0",
     "electron-packager": "^10.1.0",
-    "electron-winstaller": "2.5.2"
+    "electron-winstaller": "2.5.2",
+    "npm-run-all": "^4.1.2"
```

Comment:
This should be a proper `dependency` - the `electron-*` packages are only here because one of them (probably `electron-builder) demands to be a `devDependency`.